doc/dev/developer_guide/dash-devel.rst

```diff
@@ -1204,6 +1207,91 @@ Keep elements that affect the sentence:
   <!-- recommended -->
   <span i18n>Profile <b>foo</b> will be removed.</span>
 
+
+.. _accessibility:
+
+Accessibility
+-------------
+
+Many parts of the Ceph Dashboard are modeled on `Web Content Accessibility Guidelines (WCAG) 2.1 <https://www.w3.org/TR/WCAG21/>`_  level A accessibility conformance guidelines. 
+By implementing accessibility best practices, you are improving the usability of the Ceph Dashboard for blind and visually impaired users.
+
+Summary
+~~~~~~~
+
+A few things you should check before introducing a new code change include:
+
+1) Add `ARIA labels and descriptions <https://www.w3.org/TR/wai-aria/>`_ to actionable HTML elements.
+2) Don't forget to tag ARIA labels/descriptions or any user-readable text for translation (i18n-title, i18n-aria-label...).
+3) Add `ARIA roles <https://www.w3.org/TR/wai-aria/#usage_intro>`_ to tag HTML elements that behave different from their intended behaviour (<a> tags behaving as <buttons>) or that provide extended behaviours (roles).
+4) Avoid poor `color contrast choices <https://www.w3.org/TR/WCAG21/#contrast-minimum>`_ (foreground-background) when styling a component. Here are some :ref:`tools <color-contrast-checkers>` you can use.
+5) When testing menus or dropdowns, be sure to scan them in both opened and closed states. Sometimes issues are hidden when menus are closed.
```

Comment:
```suggestion
5) When testing menus or dropdowns, be sure to scan them with an accessibility checker in both opened and closed states. Sometimes issues are hidden when menus are closed.
```
Then link the words "accessibility checker" to the section on it you have below.